db/models.py

```diff
@@ -1,5 +1,8 @@
-from sqlalchemy import Column, Integer, String
+from sqlalchemy import Column, Integer, String, ForeignKey, Table
 from db.database import Base
+from sqlalchemy.orm import sessionmaker, relationship
```

Comment:
@algorithmike There should not be any imports in a file that are not being used.